### ISSUE TITLE
Hold shift to drop into editor?

### ISSUE THREAD

**Issue by chrmarti** [microsoft/vscode#199354]
Testing #199023

When I drag an editor over a second window, I get an "Hold shift to drop into editor" hint in the middle of the target window. Holding Shift doesn't make a difference. (Testing on Linux.)

**Comment by lramos15** [comment]
If you're using a VM it might be https://github.com/microsoft/vscode/issues/180837

**Comment by chrmarti** [comment]
I'm using Parallels.

**Comment by lramos15** [comment]
Yeah, you probably need game mode. That's what I needed when testing shift to drop in editor
https://github.com/microsoft/vscode/issues/180837#issuecomment-1522575210

**Comment by Tyriar** [comment]
I saw this too:

![image](https://github.com/microsoft/vscode/assets/2193314/cc3b8d88-9f4d-4b9b-b28a-a71602736267)

It happens consistently, but only once after doing this (if I don't drop and then try again it works fine):

1. Drag tab out to create a new window
2. Drag tab back to original window
3. Drag tab to the new window again

**Comment by bpasero** [comment]
I do not think this is floating window related, at least not that I am aware of.

**Comment by mjbvz** [comment]
Also happens if the editor fails to load (for example on file rename)

**Comment by mjbvz** [comment]
Also just to clarify the expected behavior: you should see `hold shift` when dropping a tab into a text editor. This can use used to insert resources into the file

However it should not show up for empty editor groups or editors that can't support drop (such as errored editors)
